colors/nord.vim

```diff
@@ -574,6 +574,7 @@ call s:hi("CocHintSign" , s:nord10_gui, "", s:nord10_term, "", "", "")
 
 " Neovim LSP
 " > neovim/nvim-lspconfig
+call s:hi("LspCodeLens", s:nord3_gui_bright, "", s:nord3_term, "", s:italicize_comments, "")
```

Comment:
Lets link this to [the custom `Comment` group][1] instead (append below the other LSP definitions).

```diff
- call s:hi("LspCodeLens", s:nord3_gui_bright, "", s:nord3_term, "", s:italicize_comments, "")
+ hi! link LspCodeLens Comment
```

[1]: https://github.com/arcticicestudio/nord-vim/blob/8d8b9bf8/colors/nord.vim#L276